tools/src/main/java/org/apache/kafka/tools/ToolsUtils.java

```diff
@@ -99,4 +101,26 @@ public static void prettyPrintTable(
         printRow(columnLengths, headers, out);
         rows.forEach(row -> printRow(columnLengths, row, out));
     }
+
+    public static void validateBootstrapServer(String hostPort) throws IllegalArgumentException {
+        if (hostPort == null || hostPort.isEmpty()) {
```

Comment:
Since Java 11 we can use `isBlank()` that also ignores white spaces. Unfortunately, we still need to provide compatibility with Java 8, where we can achieve the same by doing `hostPort.trim().isEmpty()`.

Reply:
Done